mopidy_alsamixer/__init__.py

```diff
@@ -22,6 +22,9 @@ def get_config_schema(self):
         schema = super(Extension, self).get_config_schema()
         schema['card'] = config.Integer(minimum=0)
         schema['control'] = config.String()
+        schema['volmin'] = config.Integer(minimum=0, maximum=100)
+        schema['volmax'] = config.Integer(minimum=0, maximum=100)
+        schema['logarithmic_volume'] = config.Boolean()
```

Comment:
That's a great suggestion. Honestly I'm a bit unsure as to whether this is logarithmic or cubic :-) I think it is cubic. It seems to work. I'd like to know which it is... I think I'll call it cubic.